File: C05_PeerCast/PeerCastCS/PeerCast/FileUtility.cs
using System.Collections.Generic;
using System.IO;
using System.Linq;

namespace PeerCast
{
    public static class FileUtility
    {
        public static List<string> GetVideoList()
        {
            List<string> videos = Directory.GetFiles(Properties.Settings.Default.FileDirectory, 
                                        "*.wmv", SearchOption.TopDirectoryOnly).ToList();
            return videos; 
        }
    }
}

File: C05_PeerCast/PeerCastVB/PeerCast/FileUtility.vb
Imports Microsoft.VisualBasic
Imports System.Collections.Generic
Imports System.IO
Imports System.Linq

Namespace PeerCast
    Public Class FileUtility
        Public Shared Function GetVideoList() As List(Of String)
            Dim videos As List(Of String) = Directory.GetFiles(My.Settings.Default.FileDirectory, "*.wmv", SearchOption.TopDirectoryOnly).ToList()
            Return videos
        End Function
    End Class
End Namespace
